src/helpers/report-builder.cjs

```diff
@@ -271,7 +271,7 @@ class ReportDetailBuilder extends ReportBuilderBase {
 	}
 
 	addDuration(duration) {
-		this._setProperty('duration', 'final', duration, { override: true });
+		this._setNestedProperty('duration', 'final', duration, { override: true });
```

Comment:
This was the actual bug. It was consuming the property and never omitting it so it was just always `0`.